web/src/components/Form/Expenses/index.js

```diff
@@ -0,0 +1,9 @@
+import CommonExpenses from './CommonExpenses';
+import ExpenseCategory from './ExpenseCategory';
+
+export default {
```

Comment:
why default and named exports for same components?

Reply:
I don't actually know.  I _think_ it's an approach I adopted once when I wanted to be able to import the parts of the module in different ways (e.g., `import { x } from './module'` and `import mod from './mod'; mod.x();`).  It's nice because you can have everything namespaced into that one imported object.  I don't feel strongly about it at all, though - I think it's a leftover habit from when I was still trying to figure out how imports/exports worked.

Reply:
i'm all for it if it makes things easier, but I'm not sure it's needed yet. perhaps let's keep them as just named exports until housing them under one object helps clean things up.

```
export { default as CommonExpenses } from './CommonExpenses'
export { default as ExpenseCategory } from './ExpenseCategory'
```